public/js/components/RightSidebar.js

```diff
@@ -48,8 +49,8 @@ function RightSidebar({ resume, command, breakOnNext,
             component: Breakpoints,
             opened: true },
           { header: "Call Stack",
-            component: () => dom.div({className: "pane-info"}, "Not Paused")
-          },
+            component: Frames,
+            componentProps: { frames }},
```

Comment:
any reason why you prefer this over `connect` in the component, which is what Breakpoints and Scopes does.